src/AvatarMember.jsx

```diff
@@ -1,6 +1,9 @@
 import React from 'react';
 import cx from 'classnames';
 import Avatar from './Avatar';
+import { getIconAsBase64Uri } from './utils/base64';
+
+const NO_PHOTO_SRC = getIconAsBase64Uri('profile');
```

Comment:
I'm interested in this choice as opposed to `<img>` with an SVG `src`? base64 strings can't be cached like other external assets, so this approach saves an http request but only in the first instance.